pkg/operation/botanist/component/etcd/bootstrap.go

```diff
@@ -112,31 +113,46 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 					Resources: []string{"events"},
 					Verbs:     []string{"create", "get", "list", "watch", "patch", "update"},
 				},
+				{
+					APIGroups: []string{corev1.GroupName},
+					Resources: []string{"serviceaccounts"},
+					Verbs:     []string{"get", "list", "watch", "create", "update", "patch", "delete"},
+				},
+				{
+					APIGroups: []string{rbacv1.GroupName},
+					Resources: []string{"roles", "rolebindings"},
+					Verbs:     []string{"get", "list", "watch", "create", "update", "patch", "delete"},
+				},
 				{
 					APIGroups: []string{corev1.GroupName, appsv1.GroupName},
 					Resources: []string{"services", "configmaps", "statefulsets"},
 					Verbs:     []string{"get", "list", "patch", "update", "watch", "create", "delete"},
 				},
+				{
+					APIGroups: []string{batchv1.GroupName},
+					Resources: []string{"jobs", "cronjobs"},
+					Verbs:     []string{"get", "list", "patch", "update", "watch", "create", "delete"},
+				},
 				{
 					APIGroups: []string{druidv1alpha1.GroupVersion.Group},
-					Resources: []string{"etcds"},
+					Resources: []string{"etcds", "etcdcopybackupstasks"},
 					Verbs:     []string{"get", "list", "watch", "update", "patch"},
 				},
 				{
 					APIGroups: []string{druidv1alpha1.GroupVersion.Group},
-					Resources: []string{"etcds/status", "etcds/finalizers"},
+					Resources: []string{"etcds/status", "etcds/finalizers", "etcdcopybackupstasks/status", "etcdcopybackupstasks/finalizers"},
 					Verbs:     []string{"get", "update", "patch", "create"},
 				},
 				{
 					APIGroups: []string{coordinationv1.GroupName},
 					Resources: []string{"leases"},
-					Verbs:     []string{"create"},
+					Verbs:     []string{"get", "list", "watch", "create", "update", "patch", "delete"},
 				},
 				{
 					APIGroups:     []string{coordinationv1.GroupName},
 					Resources:     []string{"leases"},
 					ResourceNames: []string{"druid-leader-election"},
-					Verbs:         []string{"get", "update", "patch"},
+					Verbs:         []string{"get", "update", "patch", "list"},
```

Comment:
```suggestion
					Verbs:     []string{"get", "list", "patch", "update", "watch", "create", "delete"},
```

Reply:
I removed this entire section. Since there are now full permissions on leases in general, I don't think special permissions for the `druid-leader-election` lease are needed anymore.